app/balancer/models.py

```diff
@@ -12,6 +12,9 @@ class BalanceResult(models.Model):
 
 # BalanceAsnwer is a single way to make 2 teams out of 10 players
 class BalanceAnswer(models.Model):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
To był powód błędu z wczoraj wieczorem, a raczej inne moje manipulacje w tym modelu w poprzednich commitach @PogSmok